services/release/release.go

```
@@ -109,12 +109,12 @@ func UpdateRelease(doer *models.User, gitRepo *git.Repository, rel *models.Relea
 		return err
 	}
 
+	_ = models.AddReleaseAttachments(rel.ID, attachmentUUIDs)
```

I don't think you should be ignoring error here